app/views/products/show.html.erb

```diff
@@ -161,6 +161,12 @@
 <% end %>
 
 <%= link_to 'Back', products_path(type: @type), class: 'btn btn-default' %>
+
+<style>
+  .popover {
+    max-width: 100%;
+  }
+</style>
```

Comment:
I think I would prefer to see `.popover-wide .popover { }`, something that can restrict it to the subset of the page that deals with this popover. When we add popovers to the dropdown menus this will affect those but we won't catch it till we navigate to this page.

Reply:
I confirmed, for example, if you specify an explicit id on a div and in the popover construction set the data attribute "container" to the id string the popover html will be created as a child of that container.